assembler/manual.html

```diff
@@ -461,13 +463,13 @@ <h3>3.2 SPAdes command line options </h3>
 <h4>Basic options</h4>
 <p>
     <code>-o &lt;output_dir> </code><br>
-    &nbsp;&nbsp;&nbsp;&nbsp;Specify the output directory. Required option.
+    &nbsp;&nbsp;&nbsp;&nbsp;Specify the output directory (files will be overwritten). Required option.
 </p>
 
 <a name="sc"></a>
 <p>
     <code>--sc </code><br>
-    &nbsp;&nbsp;&nbsp;&nbsp;This flag is required for MDA (single-cell) data.
+    &nbsp;&nbsp;&nbsp;&nbsp;This flag is required for MDA (single-cell) data with uneven and supposedly low coverage. Enforces <code>-k 21,33,55</code>.
```

Comment:
--sc does not enforce  k-mer length selection. User could provide any values, if necessary.

Reply:
It is mentioned elsewhere in the manual already.